vms/platformvm/upgrade/times.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package upgrade
```

Comment:
(optional) I personally feel like `fork` feels like a better name but I'm fine with `upgrade` too. We use both terms interchangeably so feel free to ignore this comment. I guess upgrade makes each upgrade sound like a strict improvement (which although is not technically true should be true haha...).